Index_v2

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link rel="stylesheet" href="version1.css">
+<link rel="stylesheet" href="version2.css">
```

Comment:
I had to change the href from version 1 to version 2, as version 2 had my css for my navigation layout. I am using css for my navigation layout. href is the Hypertext REFerence which is used in HTML code to create links to other pages.